renderer.go

```diff
@@ -0,0 +1,55 @@
+package survey
```

Comment:
moved it to core, had to make a bunch of stuff public since it is no longer in the same package.